docusaurus/docs/extensions/extensions-getting-started.md

```diff
@@ -66,13 +63,19 @@ Also for consideration:
+|          Option           | Description                                                                                                                                                                                                                                                                                   |
+| :-----------------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|     `--update \| -u`      | This will update all dependencies within the extension to match the declared versions in the shell. Update can be ran independently or along with creating an extension (e.g. `yarn create @rancher/extension my-extension --update`                                                          |
+| `--app-name \| -a <name>` | Allows specifying a different name for the skeleton application instead of using the extension name.                                                                                                                                                                                          |
+|  `--skeleton-only \| -s`  | Installs only the skeleton application without creating the extension package.                                                                                                                                                                                                                |
+|           `-l`            | This will automatically add the [`.gitlab-ci.yml`](https://github.com/rancher/dashboard/blob/master/shell/creators/extension/app/files/.gitlab-ci.yml) pipeline file for integration with GitLab                                                                                              |
+|           `-w`            | Does not add the Github workflow files [`build-extension-catalog.yml`, `build-extension-charts.yml`](https://github.com/rancher/dashboard/tree/master/shell/creators/extension/app/files/.github/workflows) to be used as Github actions. These files will be added automatically by default. |
+|           `-t`            | Does not add the template folders automatically into the Extension package. These folders will be added automatically by default                                                                                                                                                              |
+|           `-i`            | Ignores the check for the `@rancher/shell` dependency.                                                                                                                                                                                                                                        |
```

Comment:
Ah, I believe that this is something I did a while ago because of the script changes in order to have the annotations copied to `package.json` when doing a create... If I am correct, this should not be documented as it's only for internal use

Reply:
Makes sense, I can remove it.